indra/llmessage/llcoproceduremanager.cpp

```diff
@@ -47,7 +47,10 @@ static const std::map<std::string, U32> DefaultPoolSizes{
 };
 
 static const U32 DEFAULT_POOL_SIZE = 5;
-const U32 LLCoprocedureManager::DEFAULT_QUEUE_SIZE = 4096;
+// SL-14399: When we teleport to a brand-new simulator, the coprocedure queue
+// gets absolutely slammed with fetch requests. Make this queue effectively
+// unlimited.
+const U32 LLCoprocedureManager::DEFAULT_QUEUE_SIZE = 1024*1024;
```

Comment:
The biggest offenders probably aren't teleports but landmarks and gestures. Viewer needs all landmark assets loaded to let search and landmarking work and there can be thousands of landmarks in some inventories.